x-pack/plugins/security_solution/public/common/components/drag_and_drop/draggable_wrapper_hover_content.test.tsx

```diff
@@ -347,7 +363,7 @@ describe('DraggableWrapperHoverContent', () => {
     const aggregatableStringField = 'cloud.account.id';
     const draggableId = 'draggable.id';
 
-    [false, true].forEach((showTopN) => {
+    [true].forEach((showTopN) => {
```

Comment:
why remove the false, and if we're removing it why keep the forEach?

Reply:
My bad